docs/usage/macros.rst

```diff
@@ -4,44 +4,67 @@ CMake Macros
 spl_add_component
 ^^^^^^^^^^^^^^^^^
 
-Add a component to a variant.
+Add a component to the build and link it with the main project.
```

Comment:
This is not right. spl_add_component is intended to be used in a variant's parts.cmake. The target is to add it to the variant. Then it is also part of the link and the build and the reports and the unit tests and so on. And by the way: what is the "main project"? We have a Software Product Line to build the softwares of all variants of our product. I actually liked my original description.